darwin/cli_functions.py

```diff
@@ -378,8 +389,19 @@ def pull_dataset(
 
     try:
         release: Release = dataset.get_release(version)
-        dataset.pull(release=release, only_annotations=only_annotations, use_folders=folders, video_frames=video_frames)
+        dataset.pull(
+            release=release,
+            only_annotations=only_annotations,
+            use_folders=folders,
+            video_frames=video_frames,
+            force_slots=force_slots,
+        )
         print_new_version_info(client)
+        if release.format == "darwin_json_2":
+            print(f"NOTE: Your dataset has been exported using new Darwin JSON 2.0 format.")
```

Comment:
Look how `print_new_version_info` is implemented, it prints to stderr so that we can safely pipe the output we care about

Reply:
Changed!